docs/schema-cmap-instance-style-guide.md

```diff
@@ -9,18 +9,20 @@ This page provides the recommended style guidelines for creating **instance** di
 ## **Colors**
 This style guide provides a default set of colors to use, but they **can be overridden in different contexts** if needed - in such cases the legend must be updated.
 
-For **Models** colors are determined by the type of the model:
+For **Elements**, colors are determined by the type of the element:
 - `Information` - light cyan (#EDF4F6)
 - `Physical` - light blue (#96C8FF)
 - `Functional` - light green (#C8FFC8)
 - `Spatial Location` - light yellow (#FFFF96)
 - `Drawing` - purple (#C896FF)
 
-![Model colors](cmap-example/media/instance/colors-models.png)
+![Element colors](cmap-example/media/instance/colors-elements.png)
 
-All **Elements** are colored white (#FFFFFF) since element type can be inferred from their Model type.
+All **Models** are colored white (#FFFFFF) since their type can be inferred from their modeled-element.
 
-![Element colors](cmap-example/media/instance/colors-elements.png)
+![Model colors](cmap-example/media/instance/colors-models.png)
+
+On very busy instance diagrams, it may be useful to increase the thickness of the model's boundary in order to make it easier to identify.
```

Comment:
I would prefer to have clear, strict style rules rather than optional, ambiguous ones - what is "very busy"?
I don't feel like model thickness is that important - having one less rule and having all models styled consistently seems like a better option to me